src/assets/i18n/en.json

```diff
@@ -1,6 +1,8 @@
 {
-  'BUTTON': {
-    'OK': 'Ok',
-    'CANCEL': 'Cancel'
+  "BUTTON": {
```

Comment:
As long as this bundle is global, we should probably be more explicit on the label keys in order to prevent overlapping. For example, this `BUTTON` could be `SEARCH_BUTTONS`